js/index_controls.js

```diff
@@ -40,23 +40,36 @@ function calculate() {
         result.damageText = minDamage + "-" + maxDamage + " (" + minDisplay + " - " + maxDisplay + notation + ")";
         result.koChanceText = p1.moves[i].bp === 0 ? 'nice move'
                 : getKOChanceText(result.damage, p1, p2, field.getSide(1), p1.moves[i].hits, p1.ability === 'Bad Dreams');
+        var additionalText = '';
```

Comment:
recoveredText?

Reply:
Or even recoveryText

Reply:
Nope, I'm still getting familiarized with GitHub, I didn't dive into merging / squashing commits (amongst many other things).

Reply:
Fwiw  I’m not going to merge two commits, learning how to squash and at least edit multiple files on git is a good thing to learn.

Reply:
Can you close that pull request so I clean start everything back...by providing multiple files in a single PR ? :D

Reply:
Never mind ^^